testdata/configroot/scopes/global/subjects/global/rules.yml

```diff
@@ -1,36 +1,26 @@
 subject: namespace:ns
 revision: "2022"
 rules:
-        #- selector: service.name == “*”
-        #- selector: service.name == "myservice"
-- selector: target.service == "myservice.*"
+- selector: # must be empty for preprocessing adapters
```

Comment:
Not related to this PR, but we need to come up with a better way to do this. I kinda liked the `true` literal personally. I know it's a bit harder to implement with our current config resolution.